### ISSUE TITLE
Automatically focus on input field after "Ask CodeGPT"

### ISSUE THREAD

**Issue by zluka22** [timkmecl/codegpt#26]
After pressing shortcut for "Ask CodeGPT" or "Focus CodeGPT" user will immediately be able to start typing his prompt, without having to use his mouse to click on "Ask a question..." to switch focus on it.

**Comment by webventurer** [comment]
Yep I second this. At the moment I highlight code, right click and select "Ask CodeGPT". A pop up appears on the bottom right hand corner of my editor with "CodeGPT Chat: Write your question in the chat." I then have to click X and write the question in the chat. This popup serves no useful purpose and ideally there should be a setting which removes it... or just remove it!

<img width="472" alt="mapper_py_—_wordtracker-api" src="https://github.com/timkmecl/codegpt/assets/1410328/c20f7458-80a2-457b-907a-6ba818c9355d">
